js/main.js

```diff
@@ -0,0 +1,53 @@
+const NAMES_ARRAY = [
+  'Андрей','Мария',
+  'Станислав',
+  'Олег'
+];
+const COMMENTS_ARRAY = [
```

Comment:
Аналогично, чтобы все было в едином стиле:
```suggestion
const COMMENTS = [
```